hello.R

```diff
@@ -0,0 +1,26 @@
+# Hello, world!

```

Comment:
You can keep the hello.R example somewhere in your computer, but it should not go in the package.